aeneas/src/ir/VstIr.v3

```diff
@@ -90,7 +93,13 @@ class IrBuilder(ctype: Type, parent: IrClass) {
 			var ir = IrMethod.new(ctype, null, sig);
 			ir.source = m;
 			ir.facts |= Fact.M_NEW;
-			if (isVariant) ir.setFact(Fact.O_PURE | Fact.M_INLINE | Fact.V_NON_ZERO);
+			if (isVariant) {
```

Comment:
IIUC, this can remain pure, since this applies to the constructor of the type. Fields of that type might be "mutable" but values aren't.